src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,6 @@
+package nextstep.fp;
+
+public interface Conditional {
```

Comment:
```suggestion
@FunctionalInterface
public interface Conditional {
```

함수형 인터페이스임을 명시적으로 선언해 주는 것이 좋습니다.

추가로 추상 메서드가 2개 이상일 때 아래와 같이 IDE에서 빨간 줄로 에러를 미리 확인할 수도 있어요!
![image](https://github.com/next-step/java-ladder/assets/89887003/7cecee41-e4d3-4cc8-8ff4-85a84175cd2c)